core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/YouTubePlayer.kt

```diff
@@ -20,6 +20,23 @@ interface YouTubePlayer {
    */
   fun cueVideo(videoId: String, startSeconds: Float)
 
+  fun loadPlaylist(videos: List<String>, index: Int, startSeconds: Float)
```

Comment:
Please, improve existing documentation and add it where it's missing.

Describe what the method is for and all the params.